students/Chris_Hallen/lesson02/gridPrinterPart1.py

```diff
@@ -0,0 +1,19 @@
+#  Simple string manipulation
+
+row = ' - '
+space = '   '
+plus = '+'
+col = '|'
+fullRow = '' + plus + (row * 4) + plus + (row * 4) + plus
+partialRow = (col + (space * 4) + col + (space * 4) + col)
```

Comment:
these should have been caught by a linter, per pep8 vars use snake_case notation -> `partial_row`